src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -201,6 +203,8 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::pass::MVN6Decomposition>();
         manager.register_pass<ov::pass::BroadcastTransition>();
 
+        manager.register_pass<ov::pass::ConstantFolding>();
```

Comment:
I think this check requires testing on wider models scope. Have you run anything to ensure that it doesn't impact the common scenarios?

Reply:
answered above